ocamldoc/odoc_ast.ml

```diff
@@ -1022,7 +1023,7 @@ module Analyser =
    and analyse_structure_item env current_module_name loc pos_limit comment_opt parsetree_item_desc _typedtree
         table table_values =
       match parsetree_item_desc with
-      | Parsetree.Pstr_kind_abbrev _ -> Misc.fatal_error "Pstr_kind_abbrev"
+      | Parsetree.Pstr_jkind _ -> Misc.fatal_error "Pstr_kind"
```

Comment:
Do we need to plan out updating odoc to deal with abstract kinds and abbreviations? (This needn't hold up this merge of course! But I don't think this work is on the radar and maybe it should be.)